From: chrysn <chrysn@fsfe.org>
Date: Fri, 5 Apr 2013 18:54:26 +0200
Subject: Patch out verbose mode in openscad.pro

This allows having a build where debug symbols are not stripped by the
build system (and thus available for the -dbg package) without enabling
the developers' debug mechanisms as well.

Building a debug build without the flag for enabling the debug
mechanisms seems to be easier than making the build system keep the
debug symbols in a regular (`qmake CONFIG-=debug"`, as in upstream's
release scripts) build.

Forwarded: not-needed
---
 openscad.pro | 1 -
 1 file changed, 1 deletion(-)

diff --git a/openscad.pro b/openscad.pro
index 7d09a75..5234dbb 100644
--- a/openscad.pro
+++ b/openscad.pro
@@ -44,7 +44,6 @@ VERSIONDATE = 2015.04.21
 # If VERSION is not set, populate VERSION, VERSION_YEAR, VERSION_MONTH from system date
 include(version.pri)
 
-debug: DEFINES += DEBUG
 
 TEMPLATE = app
 
